extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -442,20 +451,136 @@ export abstract class AzureAuth implements vscode.Disposable {
 
 	//#region network functions
 
-	private async makeGetRequest<T>(url: string, token: string): Promise<AxiosResponse<T>> {
+	private async makeGetRequest<T>(requestUrl: string, token: string): Promise<AxiosResponse<T>> {
 		const config: AxiosRequestConfig = {
 			headers: {
 				'Content-Type': 'application/json',
-				'Authorization': `Bearer ${token}`
+				'Authorization': `Bearer ${token}`,
 			},
-			validateStatus: () => true // Never throw
+			validateStatus: () => true, // Never throw
+			// Turning off automatic proxy detection to avoid issues with tunneling agent by setting proxy to false.
+			// https://github.com/axios/axios/blob/bad6d8b97b52c0c15311c92dd596fc0bff122651/lib/adapters/http.js#L85
+			proxy: false
 		};
 
-		const response: AxiosResponse = await axios.get<T>(url, config);
-		Logger.piiSanitized('GET request ', [{ name: 'response', objOrArray: response.data?.value as TenantResponse[] ?? response.data as GetTenantsResponseData }], [], url,);
+		const httpConfig = vscode.workspace.getConfiguration("http");
+		let proxy = this.loadEnvironmentProxyValue();
+		if (!proxy) {
+			Logger.verbose("Checking workspace HTTP configuration for proxy endpoint.");
+
+			proxy = httpConfig['proxy'] as string;
+		}
+
+		if (proxy) {
```

Comment:
Do you need to handle the else condition here?

Reply:
If a proxy endpoint isn't found than we just make a request like we've always done, as seen on line 498
![image](https://github.com/user-attachments/assets/cadc7f68-c527-454d-8763-47f649035bf6)